common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1438,7 +1408,7 @@ def forum_posts_allowed(self):
 
     @property
     def number(self):
```

Comment:
I've come around on `course_number`. That's actually what it's called in Studio upon course creation. I've never called it number before - but it disambiguates it from a generic `course`, which can mean lots of things.

Just don't make me say org/course_number/run... :)